pennylane/gradients/pulse_gradient.py

```diff
@@ -571,7 +572,7 @@ def _generate_tapes_and_cjacs(tape, idx, key, num_split_times, use_broadcasting)
             split_evolve_ops = _split_evol_ops(op, word, ob.wires, tau)
             tapes.extend(_split_evol_tapes(tape, split_evolve_ops, op_idx))
     avg_prefactor = (t1 - t0) / num_split_times
-    return cjacs, tapes, avg_prefactor
+    return cjacs, tapes, avg_prefactor * prefactor
```

Comment:
The `avg_prefactor` is used to rescale the Monte Carlo integration. Here we just hijack this factor and put the rescaling factor of the generating term on top.